projects/packages/password-checker/CHANGELOG.md

```diff
@@ -5,3 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.1.0 - 2021-03-30
+### Added
+- Initial release.
+
+
```

Comment:
Hmm. How'd this happen?
```suggestion

```
I'll look into fixing changelogger to not do like this again in the future.